resources/dicts/patrols/general/medcat.json

```
@@ -350,7 +350,7 @@
                 ]
             },
             {
-                "text": "s_c laid in the patch of sun, huffing, {PRONOUN/s_c/subject} deserved a nice break, away from all {PRONOUN/s_c/poss} clanmates. {PRONOUN/s_c/subject/CAP} could always just say {PRONOUN/s_c/subject} failed to find any herbs after all.",
+                "text": "s_c laid in the patch of sun, huffing, {PRONOUN/s_c/subject} deserved a nice break, away from all {PRONOUN/s_c/poss} Clanmates. {PRONOUN/s_c/subject/CAP} could always just say {PRONOUN/s_c/subject} failed to find any herbs after all.",
```

```suggestion
                "text": "s_c laid in the patch of sun, huffing. {PRONOUN/s_c/subject/CAP} deserved a nice break, away from all {PRONOUN/s_c/poss} Clanmates. {PRONOUN/s_c/subject/CAP} could always just say {PRONOUN/s_c/subject} failed to find any herbs after all.",
```
ditto to last suggestion